vue/components/ui/atoms/container/container.vue

```diff
@@ -51,6 +77,14 @@ body.tablet .container-ripe > .container-header,
 body.mobile .container-ripe > .container-header {
     padding: 20px 15px 20px 15px;
 }
+
+.container-ripe > .container-header > .title {
+    flex: 1;
+}
+
+.container-ripe > .container-header > .header-buttons {
+    align-self: center;

```

Comment:
If this aligned center, what is the alignment of `title`? Maybe you could just `align-items` in `container-header`.